rllib/benchmarks/ppo/benchmark_atari_ppo.py

```diff
@@ -0,0 +1,123 @@
+    "ALE/Enduro-v5": (2061.1, 200000000),
+    "ALE/FishingDerby-v5": (22.6, 200000000),
+    "ALE/Freeway-v5": (29.1, 200000000),
+    "ALE/Frostbite-v5": (4141.1, 200000000),
+    "ALE/Gopher-v5": (72595.7, 200000000),
+    "ALE/Gravitar-v5": (567.5, 200000000),
+    "ALE/Hero-v5": (50496.8, 200000000),
+    "ALE/IceHockey-v5": (-11685.8, 200000000),
+    "ALE/Kangaroo-v5": (10841.0, 200000000),
+    "ALE/Krull-v5": (6715.5, 200000000),
+    "ALE/KungFuMaster-v5": (28999.8, 200000000),
+    "ALE/MontezumaRevenge-v5": (154.0, 200000000),
+    "ALE/MsPacman-v5": (2570.2, 200000000),
+    "ALE/NameThisGame-v5": (11686.5, 200000000),
+    "ALE/Phoenix-v5": (103061.6, 200000000),
+    "ALE/Pitfall-v5": (-37.6, 200000000),
+    "ALE/Pong-v5": (19.0, 200000000),
+    "ALE/PrivateEye-v5": (1704.4, 200000000),
+    "ALE/Qbert-v5": (18397.6, 200000000),
+    "ALE/RoadRunner-v5": (54261.0, 200000000),
+    "ALE/Robotank-v5": (55.2, 200000000),
+    "ALE/Seaquest-v5": (19176.0, 200000000),
+    "ALE/Skiing-v5": (-11685.8, 200000000),
+    "ALE/Solaris-v5": (2860.7, 200000000),
+    "ALE/SpaceInvaders-v5": (12629.0, 200000000),
+    "ALE/StarGunner-v5": (123853.0, 200000000),
+    "ALE/Surround-v5": (7.0, 200000000),
+    "ALE/Tennis-v5": (-2.2, 200000000),
+    "ALE/TimePilot-v5": (11190.5, 200000000),
+    "ALE/Tutankham-v5": (126.9, 200000000),
+    "ALE/Venture-v5": (45.0, 200000000),
+    "ALE/VideoPinball-v5": (506817.2, 200000000),
+    "ALE/WizardOfWor-v5": (14631.5, 200000000),
+    "ALE/YarsRevenge-v5": (93007.9, 200000000),
+    "ALE/Zaxxon-v5": (19658.0, 200000000),
+}
+
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+
+    # Compile the base command running the actual `tuned_example` script.
+    base_commands = [
+        "python",
+        "../../tuned_examples/ppo/atari_ppo.py",
+        "--enable-new-api-stack",
+        f"--num-env-runners={args.num_env_runners}" if args.num_env_runners else "",
+        f"--num-gpus={args.num_gpus}",
+        f"--wandb-key={args.wandb_key}" if args.wandb_key else "",
+        f"--wandb-project={args.wandb_project}" if args.wandb_project else "",
+        f"--wandb-run-name={args.wandb_run_name}" if args.wandb_run_name else "",
+        f"--stop-timesteps={args.stop_timesteps}",
+        f"--checkpoint-freq={args.checkpoint_freq}",
+        "--checkpoint-at-end" if args.checkpoint_at_end else "",
+    ]
+
+    # Loop through all envs (given on command line or found in `benchmark_envs` and
+    # run the `tuned_example` script for each of them.
+    for env_name in args.env.split(",") if args.env else benchmark_envs.keys():
+        # Remove missing commands.
+        commands = []
+        for c in base_commands:
+            if c != "":
+                commands.append(c)
+        commands.append(f"--env={env_name}")
+        commands.append(f"--wandb-run-name={env_name}")
+        print(f"Running {env_name} through command line=`{commands}`")
+        subprocess.run(commands)
```

Comment:
It is somehow strange to me that we emulate running from the command line which in turn runs a script that could have been triggered direclty in the loop. It makes sense to me that users can run single envs, but why not triggering them directly in the loop?